src/components/funnels/funnels/register/steps/ticket/level/card.tsx

```diff
@@ -51,23 +50,15 @@ const PriceLabel = styled.p`
 	margin-bottom: 0px !important;
 `
 
-const ExpirationNotice = styled.aside`
-	font-size: 1.2rem;
-	color: var(--color-grays-400);
-`
-
 // eslint-disable-next-line @typescript-eslint/prefer-readonly-parameter-types
-const TicketLevelCard = forwardRef(({ id, price, priceLabel, expirationDate, children, ...rest }: TicketLevelCardProps, ref: ForwardedRef<HTMLInputElement>) =>
+const TicketLevelCard = forwardRef(({ id, price, priceLabel, priceLabelDay, isDayTicket, children, ...rest }: TicketLevelCardProps, ref: ForwardedRef<HTMLInputElement>) =>
```

Comment:
Could you please undo the priceLabelDay and isDayTicket properties here, it's better to do what I mention below in en.ftl